### ISSUE TITLE
mat-error announce errors twice since it has role=alert

### ISSUE THREAD

**Issue by mertdeg2** [angular/material2#14281]
#### Bug, feature request, or proposal:
Bug

#### What is the expected behavior?
NVDA announces errors only once

#### What is the current behavior?
Errors are announced twice
![image](https://user-images.githubusercontent.com/16231118/49041159-5db0ba00-f179-11e8-8f84-cec7d9f35c66.png)

#### What are the steps to reproduce?
1. Turn on NVDA on Windows
2. Go to https://stackblitz.com/angular/brleoyyorkap?file=app%2Finput-error-state-matcher-example.ts
3. Make sure error text appears

#### What is the use-case or motivation for changing an existing behavior?
Better a11y

**Comment by crisbeto** [comment]
We are closing this issue, because we've determined, using a semi-automated process, that it is no longer relevant, has low engagement, or it does not have enough information to proceed. Feel free to comment if you think the issue should be reopened, or open a new issue with the latest version of our code instead.
